src/terraform-resource/terraform/client.go

```diff
@@ -279,6 +294,33 @@ func (c *client) Plan() (string, error) {
 	return fmt.Sprintf("%x", h.Sum(nil)), nil
 }
 
+func (c *client) JSONPlan() error {
+	// terraform show -json tfplan.binary > tfplan.json
+	planArgs := []string{
+		"show",
+		"-json",
+		fmt.Sprintf("%s", c.model.PlanFileLocalPath),
+	}
+
+	showCmd := c.terraformCmd(planArgs, nil)
+	rawOutput, err := showCmd.Output()
+	if err != nil {
+		return fmt.Errorf("Failed to retrieve output.\nError: %s\nOutput: %s", err, rawOutput)
+	}
+
+	var plan terraform_json.Plan
```

Comment:
Looks like the `terraform_json` dependency is only used as a sanity check on the output format. I'd vote for removing the dependency as I worry about being able to keep the dependency in sync. E.g. new Terraform CLI adds a new plan field but I haven't updated the terraform_json library so fields are silently missing from the output. Checking the exit code of `terraform show` feels pretty safe on its own.

Reply:
No problem. I will remove the dependency.